holidayextras/holidayextras.github.com
# Adding culture repo to projects

#### What does this PR do? (please provide any background)
- Adds culture repo to projects page to share our culture with HG readers!
- Removes the projects individual pages so instead they link straight to github. This gets rid of an un-needed step and makes HG easier to maintain (Currently we'd have to update HG projects page if summaries on github changed etc)

#### What tests does this PR have?
- None

#### How can this be tested?
- Run Hungry Geek locally
- Check Project Page works as expected
- Check new link to Culture repo exists and links to correct place

#### Screenshots / Screencast
- None

#### What gif best describes how you feel about this work?
![dog](https://media.giphy.com/media/1LweXxLwVT0J2/giphy.gif)

---

#### Reviewer 1: @lukefrake

#### Reviewer 2:
